### DISCUSSION TITLE
My campus email cannot receive the verification code from GitHub

### DISCUSSION THREAD

**Discussion by Marinhhh** [community/community#121626] [GitHub Education]
### Hi everyone,

GitHub student certification requires me to fill in the email address given by the school, but the school email cannot receive the verification information from GitHub and cannot complete the certification. I've tried adding github.com to the whitelist of my email address, but it still doesn't work. How do I complete student certification?

![image](https://github.com/community/community/assets/165647573/00d75ea9-4a56-465a-a0de-e40b20065836)
![image](https://github.com/community/community/assets/165647573/ca059514-e824-4ba0-8815-e27bbda45a40)

**Comment by haululazkiyaa** [comment]
have you try to send email to your campus email, somewhere the campus email have trouble that cant receive email from external campus domain. If you try to send email to your campus email but you didnt see anything, i recomended you to ask to your IT campus team

**Comment by davevad93** [comment]
Hi @Marinhhh ,
Have you tried checking your spam folder?
Sometimes it can happen that legitimate emails end up in spam, so it's possible that the verification email is there.

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
